controllers/main_controller.py

```diff
@@ -533,8 +498,7 @@ def __init__(self, *args, **kw):
         self._cache_expiration = 60 * 60 * 24 * 7
 
     def _render(self, *args, **kw):
-        path = os.path.join(os.path.dirname(__file__), "../templates/apiwrite.html")
-        return template.render(path, self.template_values)
+        return jinja2_engine.render('apiwrite.html', self.template_values)
 
 
 class MatchInputHandler(CacheableHandler):
```

Comment:
Can we kill this altogether? This mentions being for the 2015 Championship - feels like we might not need this anymore?

Reply:
yeah, this can go